lib/screens/v2/candidate_generator/widgets/cr_departures.ex

```diff
@@ -86,4 +92,27 @@ defmodule Screens.V2.CandidateGenerator.Widgets.CRDepartures do
 
     Departure.fetch(params, Keyword.new(opts))
   end
+
+  defp fetch_last_schedule_tomorrow(direction_to_destination, station, now) do
+    tomorrow =
+      now
+      |> Timex.to_datetime("America/New_York")
+      |> Timex.shift(days: 1)
+      |> Timex.format!("{YYYY}-{0M}-{0D}")
```

Comment:
So initially, I thought that this would not be an issue because we would be showing schedules for the next day. But I don't think that's true. I think we start showing the next day's schedules when the service day changes, which I believe is at 3AM. Regardless, I changed the logic so that early AM does not shift to tomorrow. This seems a little hacky but I think it should be reliable. Let me know what you think.

Reply:
I think this makes sense and I agree is more reliable. I guess there's some technicality when it comes to the verbiage being used, but it's probably not a big deal. Like if we begin saying the last train into Boston tomorrow is at [x] time around 3AM then _technically_ tomorrow is referring to the next calendar day, and not just later on today, which is what I think we mean. But this might be overthinking things haha.

Reply:
I looked into the API source code and found that the default date for the `schedules` endpoint is the current service day. So when we query at 2AM, we are still trying to get schedules from the day before. After 3AM, we start showing schedules for the next service day. With that in mind, I simplified the logic a bit.

As for the wording, I didn't even think about that. I will bring this up to Mindy and see what she has in mind for the early hours.

Reply:
Had a chat with PaulH and Mindy and they are ok with saying "Tomorrow" even if it is midnight-3AM. There were some wording changes needed, though. Biggest change is the removal of the config. The final wording on these screens is similar enough that I can build it based on if the train is inbound/outbound.

Reply:
Sounds good to me!

Let me just double check I'm following the new logic correctly: So effectively, until  ~6AM on a given overnight period we would be showing the last train on the next **service day**. And then I guess at that point the night is over so we would switch to a daytime state for the CR widget?

Reply:
Correct, except it should only be until 3AM. The service day changes at 3AM, so starting then we will starting showing schedules again.

Reply:
Gotcha. Makes sense!